### PR TITLE
Rafael

### PR DESCRIPTION
**Pull request by bethaile**
Hi Rafael I would like to pull from your branch

### CODE REVIEW

File: ProofOfConcept/src/main/java/edu/mum/cs544/wind/domain/Person.java [hunk 1]
```diff
@@ -40,6 +40,10 @@ public Person() {
         roles = new HashSet<>();
         sessions = new ArrayList<>();
     }
+    
+    public Person(long id) {
```
Comment (spham1412) [inline ProofOfConcept/src/main/java/edu/mum/cs544/wind/domain/Person.java:44]: @rafaelmum why do we have this constructor for the id?

File: ProofOfConcept/src/main/java/edu/mum/cs544/wind/domain/Person.java [hunk 2]
```diff
@@ -54,7 +58,7 @@ public long getId() {
         return id;
     }
 
-    private void setId(long id) {
+    public void setId(long id) {
```
Comment (spham1412) [inline ProofOfConcept/src/main/java/edu/mum/cs544/wind/domain/Person.java:61]: @rafaelmum and why do you change this setId() to public, we shouldn't be able to change the id once it's generated, that's why I put it private.

**Comment by rafaelmum** [comment]
Because it was necessary to create a Session, I was getting an exception,
so I add this constructor and resolved. We need to inform the conselour
when creating a Session, but just the ID on JSON. If you know how to solve
this using another way please tell us, but it is OK now.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Livre
de vírus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>.
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by rafaelmum** [comment]
I do not remember why I change this, I think I was trying to solve the
problem with creating Session, I think is OK to put back to private, but I
do not think this is important anyway..

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Livre
de vírus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>.
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Pull request merged**